XML/TestCases/CommunityTests.xml

```diff
@@ -6,7 +6,7 @@
         <setupType>OneVM1Disk</setupType>
         <TestParameters>
             <param>FSTYP=xfs</param>
-            <param>TEST_DEV=/dev/sdc1</param>
+            <param>TEST_DEV=</param>
```

Comment:
remove this from the xml, include it as part of the script.
No need to have an empty variable defined here.